ctapipe/image/muon/intensity_fitter.py

```diff
@@ -194,6 +191,10 @@ def image_prediction(
         max_lambda_m=max_lambda.to_value(u.m),
     )
 
+# Define numba vectorized gaussian cdf
+@vectorize([double(double, double, double)])
+def gaussian_cdf(x, mu, sig):
+    return 0.5*(1 + erf((x-mu)/(np.sqrt(2*sig**2))))
```

Comment:
Please use the black formatting pre-commit hook, see https://cta-observatory.github.io/ctapipe/getting_started/index.html